coffee-module/coffee-module-redisstream/src/test/java/hu/icellmobilsoft/coffee/module/redisstream/StreamPublisherTest.java

```diff
@@ -0,0 +1,105 @@
+/*-
+ * #%L
+ * Coffee
+ * %%
+ * Copyright (C) 2020 - 2022 i-Cell Mobilsoft Zrt.
+ * %%
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ * #L%
+ */
+package hu.icellmobilsoft.coffee.module.redisstream;
+
+import hu.icellmobilsoft.coffee.module.redisstream.annotation.RedisStreamProducer;
+import hu.icellmobilsoft.coffee.module.redisstream.config.StreamGroupConfig;
+import hu.icellmobilsoft.coffee.module.redisstream.publisher.RedisStreamPublisher;
+import org.jboss.weld.junit5.EnableWeld;
+import org.jboss.weld.junit5.WeldInitiator;
+import org.jboss.weld.junit5.WeldJunit5Extension;
+import org.jboss.weld.junit5.WeldSetup;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Tag;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+
+import javax.enterprise.context.RequestScoped;
+import javax.inject.Inject;
+import java.lang.reflect.Field;
+
+/**
+ * Test for RedisStream Publisher class
+ *
+ * @author peter.kovacs
+ * @since 1.11.0
+ */
+@EnableWeld
+@Tag("weld")
+@ExtendWith(WeldJunit5Extension.class)
+@DisplayName("Redis Publisher tests")
+class StreamPublisherTest {
```

Comment:
https://github.com/i-Cell-Mobilsoft-Open-Source/coffee/blob/master/coffee-module/coffee-module-redis/src/test/java/hu/icellmobilsoft/coffee/module/redis/config/RedisConfigTest.java

egy ehhez hasonlo teszt is kellene, ami a StreamGroupConfig osztalyt veszi celba